src/components/sidebar/icon-contrast-ratio.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+import * as ReactTooltip from 'react-tooltip';
+import Language from '../../language/norwegian';
+import './icon-contrast-ratio.less';
+
+var Line = require('rc-progress').Line; // tslint:disable-line
```

Comment:
Litt overkill med en npm-avhengighget for å lage en farget boks imho

Reply:
Gode poeng :)